pkg/compactor/block_upload_test.go

```diff
@@ -303,6 +303,22 @@ func TestMultitenantCompactor_StartBlockUpload(t *testing.T) {
 			},
 			expBadRequest: fmt.Sprintf("version must be %d", block.TSDBVersion1),
 		},
+		{
+			name:            "invalid version",
```

Comment:
Copy pasta from above test case?

Reply:
busted! :) that's what we get when using slice instead of map of name => testcase.